src/panels/events.js

```diff
@@ -44,33 +31,32 @@ var EventsPanel = React.createClass({
         setInterval(this.updateEvents, EVENTS_INTERVAL_MS);
     },
 
-    formatDate: function(date, isEndDate, showDate) {
-        var jsDate = date.toJSDate();
-        if (isEndDate && date.isDate) {
-            jsDate.setDate(jsDate.getDate() - 1);
-        }
+    formatDate: function(date, isEndDate, showDate, isAllDay) {
         var dateComponents = [];
         if (showDate) {
-            dateComponents.push(moment(jsDate).format('MMM D'));
+            dateComponents.push(date.format('MMM D'));
         }
-        if (!date.isDate) {
-            dateComponents.push(time.formatTime(jsDate));
+        if (!isAllDay) {
+            dateComponents.push(date.format('h:mm a'));
```

Comment:
We should continue to use `time.formatTime` here because of formatting issues with the Raspberry Pi when using moment. See [this comment](https://github.com/IllinoisWCS/wcs-display/blob/master/src/utils/time.js) for details.

![image](https://cloud.githubusercontent.com/assets/424214/19841969/73783314-9ee1-11e6-9fb9-c8fd2637c8c7.png)